Common.props

```diff
@@ -11,8 +11,9 @@
 
   <!-- Build with a common platform version -->
   <PropertyGroup Label="Globals">
-    <!--Use RS1 SDK for compatibility with Desktop Bridge debut and build agent-->
-    <WindowsTargetPlatformVersion Condition="'$(WindowsTargetPlatformVersion)'==''">10.0.14393.0</WindowsTargetPlatformVersion>
+    <!--Use RS2 SDK for compatibility with Desktop Bridge-->
+    <!--Using RS2 SDK to prevent breakaway with powershell scripts-->
+    <WindowsTargetPlatformVersion Condition="'$(WindowsTargetPlatformVersion)'==''">10.0.15063.0</WindowsTargetPlatformVersion>

```

Comment:
>15063 [](start = 89, length = 5)

RS2? We need a breaking change notification if this is intentional.

Reply:
How do we do that?